main/guides/getting-started/README.md

```diff
@@ -15,7 +15,7 @@ git clone -b community-dev https://github.com/Agoric/agoric-sdk # "community-dev
 cd agoric-sdk
 yarn install
 yarn build
-yarn link-cli ~/bin/agoric # creates an Agoric directory to reference
+yarn link-cli ~/bin/agoric # creates an executable script for PATH reference
```

Comment:
I'm not sure what "for PATH" is supposed to mean here.

```suggestion
yarn link-cli ~/bin/agoric # creates an executable script
```

We discuss `$PATH` in the expanded docs later in the page. The audience for this quickstart section is folks that already know norms such as PATH.